### PR TITLE
Travis CI: Remove the redundant tests

### PR DESCRIPTION
**Pull request by cclauss**
Sorry about that!  Now it is two test runs (Python 2 and Python 3) instead of four.

### CODE REVIEW

**Comment by malwaredllc** [comment]
Awesome, this seems to be catching bugs pretty well and you seem to be doing a good job fixing them, so thank you. Let me know when you've done the last commit and I'll merge this.

File: byob/modules/ransom.py [hunk 1]
```diff
@@ -15,6 +16,7 @@
 # packages
 import Crypto.PublicKey.RSA
 import Crypto.Cipher.PKCS1_OAEP
```
Comment (malwaredllc) [inline byob/modules/ransom.py]: I just noticed, `Crypto.Cipher.PKCS1_OAEP` needs to updated to `Cryptodome.Cipher.PKCS1_OAEP` - if you could integrate this into your next commit I would appreciate it.

File: byob/modules/ransom.py [hunk 2]
```diff
@@ -15,6 +16,7 @@
 # packages
 import Crypto.PublicKey.RSA
```
Comment (malwaredllc) [inline byob/modules/ransom.py]: I just noticed, `Crypto.PublicKey.RSA` needs to updated to `Cryptodome.PublicKey.RSA` - if you could integrate this into your next commit I would appreciate it.

**Comment by cclauss** [comment]
__You need to carefully review each change.__  I am _making guesses_ based on previous experience but I do not know this codebase _at all_!  Some of my guesses might well be wrong.

**Comment by malwaredllc** [comment]
> https://travis-ci.com/malwaredllc/byob/jobs/143508615#L594-L596 you might want del() instead of delete(). However deleting sys.argv[0] does not sound like a great idea to me.

The `delete()` function is a utility that is part of the `util` module - it attempts to delete files from the host filesystem in different ways until it is successful. 

The line `delete(sys.argv[0])` is used by the payload to self-destruct if the server gives the command to abort.

**Pull request merged**